gax-grpc/src/main/java/com/google/api/gax/grpc/GrpcCallableFactory.java

```diff
@@ -149,6 +150,16 @@ private GrpcCallableFactory() {}
       ClientContext clientContext) {
     UnaryCallable<RequestT, ResponseT> callable =
         createBaseUnaryCallable(grpcCallSettings, batchingCallSettings, clientContext);
+
+    // NOTE: Since batching happens asynchronously and the outermost callable simply buffers the
+    // request. Tracing will only start on the inner callable that accepts the batch.
+    callable =
```

Comment:
Minor: callable is reassined here in an odd way, please either give it two different names, or just call `createBaseUnaryCallable()` directly as first argument of `TracedBatchingCallable` constructor here.